RecoEgamma/Configuration/python/RecoEgamma_cff.py

```diff
@@ -6,6 +6,8 @@
 from RecoEgamma.EgammaPhotonProducers.conversionSequence_cff import *
 from RecoEgamma.EgammaPhotonProducers.conversionTrackSequence_cff import *
 from RecoEgamma.EgammaPhotonProducers.allConversionSequence_cff import *
+from RecoEgamma.EgammaPhotonProducers.conversionOpenTrackSequence_cff import *
```

Comment:
These imports must be moved in Configuration/StandardSequences/python/Reconstruction_cff.py